include/deal.II/base/exceptions.h

```diff
@@ -1512,9 +1512,10 @@ namespace deal_II_exceptions
 #ifdef DEAL_II_HAVE_BUILTIN_EXPECT
 #  define AssertThrow(cond, exc)                                       \
     {                                                                  \
-      if (__builtin_expect(!(cond), false))                            \
+      if (!(cond))                                                     \
```

Comment:
Why did you change this?

Reply:
This is not necessary. I copied the text from Assert. I removed it.